js/securesubmit/checkout-form.js

```diff
@@ -711,6 +856,15 @@ document.observe('dom:loaded', function () {
                         }
                     }
                 });
+
+                if (window.amastyCompleteCheckoutOriginal && document.querySelector('#amscheckout-onepage')) {
+
+                    document.querySelector('#ss-banner').style.backgroundSize = '325px 40px';
+                    document.querySelector('#cc-number').className = 'amasty_one_page_checkout';
+                    document.querySelector('#expiration-date').className = 'amasty_one_page_checkout';
+                    document.querySelector('#ccv').className = 'amasty_one_page_checkout';
```

Comment:
@ttnnkkrr Two points here:

- `document.querySelector('#node-id')` should really be `document.getElementById('node-id')` since nodes should not share id's
- Each of the above four lines should verify the nodes exist first, e.g.:

```javascript
var banner = document.getElementById('ss-banner');
if (banner) {
    banner.style.backgroundSize = '325px 40px';
}
```